profiles/common/themes/ec_resp/template.php

```diff
@@ -1079,6 +1079,11 @@ function ec_resp_menu_link__menu_breadcrumb_menu(array $variables) {
     }
   }
 
+  if (theme_get_setting('enable_interinstitutional_theme')) {
+    $element['#title'] = 'EUROPA';
+    $element['#href'] = 'http://europa.eu/index_en.htm';
+  }
```

Comment:
Don't output the title in the HTML in all-caps. The all caps is a styling issue, so it should be done with CSS  text-transform.